e2e/scenario_test/settings/not_set/basal_rates/settings.no_basal_rates.spec.js

```diff
@@ -0,0 +1,51 @@
+const { setup, pump, cgm, match, settings } = require('../../../../../src/index');
+
+describe('Pump Settings', () => {
+    beforeAll(async () => {
+        await setup.LaunchLoop();
+        await setup.LoadScenarios(device.id);
+        await cgm.Add();
+        await pump.Add();
+    });
+    describe('Closed loop is not allowed', () => {
+        describe('When basal rates are not set', () => {
+            beforeAll(async () => {
+                await setup.LoadScenario('flat_cgm');
+            });
+            it('should NOT set the suspend threshold', async () => {
+                await settings.Suspend('65');
+            });
+            it('should have correction range set', async () => {
+                await settings.CorrectionRanges([{time: '12:00 AM', min:'179',max:'180'}]);
+            });
+            it.skip('should set the basal rates', async () => {
+                await settings.BasalRates('0.1');
+            });
+            it('should set the delivery limits', async () => {
+                await settings.DeliveryLimits('0.5', '10.0');
+            });
+            it('should set the insulin model', async () => {
+                await settings.InsulinModel(settings.InsulinModels.RapidAdults);
+            });
+            it('should set the carb ratios', async () => {
+                await settings.CarbRatios('8');
+            });
+            it('should set insulin sensitivites set', async () => {
+                await settings.InsulinSensitivities('500');
+            });
+            it('should toggle on closed loop', async () => {
+                await settings.ClosedLoop();
+            });
+            it('should not be in closed loop mode', async () => {
+                await expect(match.loop.Icon()).toHaveLabel('Waiting for first run');
+            });
+            it('should show error that indicates why not in closed loop mode', async () => {
+                await match.loop.Icon().tap();
+                await waitFor(match.accessible.AlertLabel('Missing Data: Insulin Effects')).toExist().withTimeout(2000);
```

Comment:
I'm confused. What is the test actually testing? What setting is not set?

Reply:
see above - i skipped the setting of the basal rates. Test shows that because of this there is an alert when we check the loop status